account_move_tier_validation_approver/tests/test_account_move_tier_validation_approver.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 ForgeFlow (http://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountMoveTierValidationApprover(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.res_partner_1 = self.env["res.partner"].create(
+            {"name": "Wood Corner", "email": "example@yourcompany.com"}
+        )
+        self.product_1 = self.env["product.product"].create(
+            {"name": "Desk Combination"}
+        )
+        self.currency_usd = self.env["res.currency"].search([("name", "=", "USD")])
+        self.test_user_1 = self.env["res.users"].create(
+            {"name": "User", "login": "test1", "email": "example@yourcompany.com"}
+        )
+        self.test_approver = self.env["res.users"].create(
+            {"name": "Approver", "login": "test2", "email": "example@yourcompany.com"}
+        )
```

Comment:
```suggestion

from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT


class TestAccountMoveTierValidationApprover(TransactionCase):
    def setUp(self):
        super().setUp()
        self.env = self.env(context=dict(self.env.context, **DISABLED_MAIL_CONTEXT))
        self.res_partner_1 = self.env["res.partner"].create(
            {"name": "Wood Corner", "email": "example@yourcompany.com"}
        )
        self.product_1 = self.env["product.product"].create(
            {"name": "Desk Combination"}
        )
        self.currency_usd = self.env["res.currency"].search([("name", "=", "USD")])
        self.test_user_1 = self.env["res.users"].create(
            {"name": "User", "login": "test1", "email": "example@yourcompany.com"}
        )
        self.test_approver = self.env["res.users"].create(
            {"name": "Approver", "login": "test2", "email": "example@yourcompany.com"}
        )
```

Odoo is trying to sent emails for the users created here.
It can be made more efficient by using DISABLED_MAIL_CONTEXT

Like done in sale_tier_validation module
https://github.com/OCA/sale-workflow/blob/17.0/sale_tier_validation/tests/test_tier_validation.py#L6-L31